packages/react-dev-utils/detectPort.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
```

Comment:
Yeah I was going to ask you about the licensing before merging. It's MIT.

Serve uses `detect-port`.